plugins/inventory/cobbler.py

```diff
@@ -44,6 +44,15 @@
         type: list
         default: []
         elements: str
+      include_profiles:
+        description:
+          - Profiles to include from inventory.
+          - If specified, all other profiles will be excluded.
+          - I(exclude_profiles) is ignored if I(include_profiles) is specified.
```

Comment:
Maybe also add a paragraph like this to the `exclude_profiles` description (like `If I(include_profiles) is specified, this option is ignored.`).